lib/offsite_payments/integrations/realex.rb

```diff
@@ -118,7 +118,7 @@ def initialize(order, account, options = {})
           # however it does echo any other param so we send it twice.
           add_field 'X-CURRENCY', @currency
           add_field 'X-TEST', @test.to_s
-          add_field 'ORDER_ID', order + @timestamp.to_i.to_s
+          add_field 'ORDER_ID', order.to_s + @timestamp.to_i.to_s
```

Comment:
line is getting a bit long - I think the second parameter could be more readable as `"#{order}#{@timestamp.to_i}"` but that's just a preference